src/main/java/net/io/kino/model/Order.java

```diff
@@ -0,0 +1,78 @@
+package net.io.kino.model;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonIdentityReference;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+public class Order implements Comparable<Order>
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    @OneToMany(
+            mappedBy = "order",
+            cascade = CascadeType.ALL,
+            orphanRemoval = true
+    )
+    private List<Ticket> tickets;
+    @Transient
```

Comment:
To usunięcie z serializacji to jest obejście jakiegoś problemu?
W Ticket robicie, żeby zwracało id. Tutaj nie będzie to też ok? (nie testowałem)